src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,48 +1,43 @@
 package edu.kis.vh.nursery;
 
-public class DefaultCountingOutRhymer {
-
-
-    private static final int ERR = -1;
-    private static final int INITVALUE = -1;
-    private static final int MAXARRAY = 12;
-
-    private int[] numbers = new int[MAXARRAY];
+import edu.kis.vh.nursery.bridgeDesign.CollectionHierarchy;
+import edu.kis.vh.nursery.bridgeDesign.IntArrayStack;
+import edu.kis.vh.nursery.bridgeDesign.IntLinkedList;
 
-    private int total = INITVALUE;
+public class DefaultCountingOutRhymer {
 
-    public int getTotal() {
-        return total;
+    private CollectionHierarchy intArrayStack=new IntArrayStack();
+    public DefaultCountingOutRhymer(){}
+    public DefaultCountingOutRhymer(IntArrayStack intArrayStack) {
+        this.intArrayStack = intArrayStack;
+    }
+    public DefaultCountingOutRhymer(IntLinkedList intLinkedList) {
+        this.intArrayStack = intLinkedList;
     }
+//    public int getTotal() {
+//        return intArrayStack.getTotal();
+//    }
```

Comment:
Such comments should be removed, however the method was added to the rhymer in the previous lab and should remain.